calico_containers/calico_ctl/container.py

```diff
@@ -235,8 +235,12 @@ def container_remove(container_name):
                 # enforcing assignments strictly in datastore.py.
                 client.unassign_address(pool, ip)
 
-    # Remove the endpoint
-    netns.remove_veth(endpoint.name)
+    try:
+        # Remove the interface if it exists
+        netns.remove_veth(endpoint.name)
+    except CalledProcessError:
+        print "Could not remove calico interface"
```

Comment:
Capital C.

Put in the interface name otherwise there is not much to go on to help debug.